src/qt/peertablemodel.cpp

```diff
@@ -154,30 +154,7 @@ void PeerTableModel::refresh()
         new_peers_data.append(stats);
     }
 
-    // Handle peer addition or removal as suggested in Qt Docs. See:
```

Comment:
removing this reintroduces #160 as described here: https://github.com/bitcoin-core/gui/issues/160#issuecomment-855515389